examples/example_config_files/test_detector.yaml

```diff
@@ -0,0 +1,84 @@
+name: TestDetector
+units:
+  length: mm
+  angle: deg
+  potential: V
+  temperature: K
+grid:
+  coordinates: cartesian
+  axes:
+    x:
+      from: -9
+      to: 9
+      boundaries: inf
+    y:
+      from: -9
+      to: 9
+      boundaries: inf
+    z:
+      from: -9
+      to: 9
+      boundaries: inf
+medium: vacuum
+detectors:
+  - translate: 
+      x: 0
+    rotate: 
+      Z: 0
+    bulk:
+      material: HPGe
+      temperature: 78
+      impurity_density:
+        name: linear
+        x:
+          init: 0
+          gradient: 0
+        y:
+          init: 0
+          gradient: 0
+        z:
+          init: 10000000
+          gradient: 50000
+      charge_drift_model:
+        include: ADLChargeDriftModel/drift_velocity_config.json
```

Comment:
Did you test this (to read in a `json` file with the `yaml` parser?

Reply:
I did not thought about that, but yes, it works.